wiki/Chat_Console/tr.md

```diff
@@ -0,0 +1,107 @@
+| ![Player is submitting a beatmap](img/Chat_Console-Submitting.png "Oyuncu bir beatmap gönderiyor") | Turkuaz - Yaptıkları bir beatmapi gönderiyor (yükleme veya güncelleme). |
+| ![Player is modding a beatmap](img/Chat_Console-Modding.png "Oyuncu bir beatmap modluyor") | Yeşil - Bir başkasının beatmapini modluyor ya da düzenliyor. |
+| ![Player in Multiplayer](img/Chat_Console-Multiplayer.png "Oyuncu Çoklu'da") | Kahverengi - Kullanıcı Çoklu modda, ama oynamıyor. |
+| ![Oyuncu Multiplayer oynuyor](img/Chat_Console-Multiplaying.png "Oyuncu Çoklu'da oynuyor") | Sarı - Şu anda Çoklu'da oynuyor. |
+| ! [Player is AFK](img/Chat_Console-Afk.png "Oyuncu AFK") | Siyah - Aktif değil ya da klavyeden uzakta (AFK). |
+| ![Player is connected via IRC](img/Chat_Console-IRC.png "Oyuncu IRC üzerinden bağlıdır") | İçeriksiz Lacivert - Oyuncu oyun içinde değildir, ancak bir IRC istemcisine bağlıdır ya da istatistikler mevcut değildir. |
+
+Herhangi bir kullanıcı paneline tıkladığınızda bir seçenek ekranı açılır.
+
+![User panel options](img/Chat_Console-Userpaneloptions.jpg "Kullanıcı paneli seçenekleri")
+
+Etkinleştirmek için numarayı tıklatın veya çubuğu tıklatın:
+
+1. `İzlemeye Başla`: Eğer kullanıcı bir beatmap oynuyorsa ve siz beatmape sahipseniz, onları oynarken izleyebilirsiniz. Adınız Seyirciler listesinde yer alacak.
+2. `Profili Göster`: Oyuncunun profilinin Web sayfası tarayıcınızda açılır.
+3. `Sohbeti Başlat`: Kullanıcıyla özel sohbet sekmesini açar.
+4. `Oyuna davet et`: (Çoklu odadaysanız) Kullanıcıdan odanıza gelmesini isteyin.
+5. `Arkadaş Olarak Ekle (Arkadaşlıktan Kaldır)`: Kullanıcıyı arkadaş listenize ekler (veya listenizden kaldırır)
+6. `Kullanıcıyı Bildir...`: Kullanıcıyı kötü davranış için bildirin. Aksi belirtilmedikçe asla kullanılmamalıdır. Bir kullanıcıyı oyun içinde, ya da forumlarda olduğu gibi web'de, belli birkaç şey için raporlayabilirsiniz,
+7. `Kullanıcıyı Sustur`: Kullanıcı tarafından yapılan herhangi bir sohbet gönderisi sizin sohbet konsolunuzda gözükmeyecektir.
+8. `Kapat`: Paneli kapatır.
+
+## Komut listesi
+
+### /help
+
+| Komut | Etki | Örnek | BanchoBot yanıtı |
+| :-- | :-- | :-- | :-- |
+| `/addfriend [kullanıcı]` | Arkadaş listenize `[kullanıcı]` ekleyin. | `/addfriend Amigo` | Artık Amigo ile arkadaşsınız. |
+| `/delfriend [kullanıcı]` | Arkadaş listenizden `[kullanıcı]` kaldırın. | `/delfriend Amigo` | Artık Amigo ile arkadaş değilsiniz. |
+| `/away [mesaj]` | Bir dışarıda iletisi ayarlar (Size PM gönderen kullanıcılara gönderilir). İptal etmek için iletiyi boş bırakın. | `/away Ben John Smith.` | You have been marked as being away: Ben John Smith. Amigo size /msg John Neredesin~? dediğinde BanchoBot: Ben John Smith. |
+| `/bb` | BanchoBot'a bir komutu doğrudan göndermenizi sağlar. | `/bb !faq tr:peppy` | peppy, osu!'nun kurucusu ve baş geliştiricisidir. Projelerin çoğunu kendisi yönetiyor. |
```

Comment:
the English document was written differently here. perhaps we should change this, although `rank #` command doesn't seem to work, but `!stats [user]` do